pcdsdevices/cvmi_motion.py

```diff
@@ -41,6 +46,15 @@ class CVMI(BaseInterface, GroupDevice):
 
     sample_paddle = Cpt(BeckhoffAxis, ':MMS:07', kind='normal')
 
+    # LEDs
+    led1_name = Cpt(EpicsSignal, ':LED:01:NAME', kind='normal')
```

Comment:
```suggestion
    led1_name = Cpt(EpicsSignal, ':LED:01:NAME', kind='normal', string=True)
```
You'll need to add this here and the other `NAME` one.

The reason is that pytmc + EPICS under the hood represent these strings as arrays of 8-bit characters, and we need to tell ophyd via pyepics how to interpret what it sees.